packages/itwinui-react/src/core/Table/Table.test.tsx

```diff
@@ -2040,7 +2040,7 @@ it('should render sub-rows and handle expansions', async () => {
   expect(onExpand).toHaveBeenNthCalledWith(1, [data[0]], expect.any(Object));
   expect(onExpand).toHaveBeenNthCalledWith(
     2,
-    [data[0], data[0].subRows[1]],
+    [data[0], data[1]],
```

Comment:
Why was this changed? I would expect sub-rows to work the same as before.

Reply:
I should have addressed and asked about this in the previous PR. This was how it originally worked before the separating sub-component PR and I was planning to fix this issue after attempting to transform the sub-component to be sub-row (as the previous PR was the continuation of this one). After I got the sub-component to work as a sub-row, it went back to the original behavior. I'm assuming it's because how we conditionally render the main row/sub-row but I'm not entirely sure about this.

Reply:
Ah ok, that makes sense. So it's the same as it was before #2240.